xarray/tests/test_plot.py

```diff
@@ -636,17 +636,24 @@ def test_xy_strings(self):
         self.assertEqual('x', ax.get_ylabel())
 
     def test_positional_coord_string(self):
-        with self.assertRaisesRegexp(ValueError, 'cannot supply only one'):
-            self.plotmethod('y')
-        with self.assertRaisesRegexp(ValueError, 'cannot supply only one'):
-            self.plotmethod(y='x')
+        self.plotmethod(y='x')
+        ax = plt.gca()
+        self.assertEqual('x', ax.get_ylabel())
+        self.assertEqual('y', ax.get_xlabel())
+
+        self.plotmethod(x='x')
+        ax = plt.gca()
+        self.assertEqual('x', ax.get_xlabel())
+        self.assertEqual('y', ax.get_ylabel())
 
     def test_bad_x_string_exception(self):
-        with self.assertRaisesRegexp(ValueError, 'x and y must be coordinate'):
+        with self.assertRaisesRegexp(ValueError, 'x and y must be coordinate variables'):
             self.plotmethod('not_a_real_dim', 'y')
+        with self.assertRaisesRegexp(ValueError, 'x must be a dimension name if y is not supplied'):
```

Comment:
I don't have a character count handy, but is this at 80 or more characters? If so, please split onto another line. I think flake8 or pep8 should catch this.